Automattic/wp-calypso
# Hosting Metrics: Clone existing log interface into Site Monitoring context

See https://github.com/Automattic/dotcom-forge/issues/3388
See https://github.com/Automattic/dotcom-forge/issues/3393

## Proposed Changes

Introduces 'PHP Logs' and 'Webserver Logs' into the Site Monitoring context:

<img width="1819" alt="image" src="https://github.com/Automattic/wp-calypso/assets/36432/5c785315-c993-4ab5-bca3-00749dee90d8">

## Testing Instructions

1. Navigate to the existing Site Logs page and verify everything still works as expected: `/site-logs/<site>`
2. Navigate to the Site Monitoring page and verify you can access logs: `/site-monitoring/<site>`
3. Downloading logs should work as expected in both contexts.
4. Filtering by dates should work as expected in both contexts.
